src/main/java/edu/wpi/grip/core/sources/CameraSource.java

```diff
@@ -173,13 +173,13 @@ private synchronized void startVideo(FrameGrabber grabber) throws IOException {
                     // TODO Pass Exception to the UI.
                     System.err.println("Webcam Frame Grabber Thread crashed with uncaught exception:");
                     exception.printStackTrace();
-                    eventBus.post(new FatalErrorEvent(exception));
+                    eventBus.post(new UnexpectedThrowableEvent(exception));
                     try {
                         stopVideo();
                     } catch (TimeoutException e) {
                         System.err.println("Webcam Frame Grabber could not be stopped!");
                         e.printStackTrace();
```

Comment:
Events should not be used to log. Logging is a separate thing that adds overhead.